### ISSUE TITLE
Sit missing from lts-19.0

### ISSUE THREAD

**Issue by juhp** [commercialhaskell/lts-haskell#336]
Unfortunately `Sit` had to be temporarily disabled, due to this Stackage build problem:
```
Preprocessing library for Sit-0.2021.1.18..
Cabal-simple_mPHDZzAJ_3.4.1.0_ghc-9.0.2: The program 'happy' is required but
it could not be found
```
Likely something to do with the initial stack build bootstrapping environment.

We can add it back for lts-19.1 if possible

cc @andreasabel

**Comment by andreasabel** [comment]
Fixed by releasing 0.2022.3.18.

**Comment by juhp** [comment]
Okay, sure no worries - I will close this out now.
But feel free to add it back later of course if you wish.

**Comment by andreasabel** [comment]
> Fixed by releasing 0.2022.3.18.

Can you add this version back in into LTS?

**Comment by alexeyzab** [comment]
Added in [LTS-19.5](https://www.stackage.org/lts-19.5/package/Sit-0.2022.3.18).
